addons/stock/models/stock_move_line.py

```diff
@@ -521,7 +521,7 @@ def _free_reservation(self, product_id, location_id, quantity, lot_id=None, pack
             move_to_recompute_state = self.env['stock.move']
 
             rounding = self.product_uom_id.rounding
```

Comment:
Now this is the only reason for this not being an `@api.model` type method. Make it part of the interface?

Reply:
Implemented in: @blaggacao
[REF] stock: Reuse free reservation method